google-apis-core/lib/google/apis/core/api_command.rb

```diff
@@ -18,7 +18,7 @@
 require 'google/apis/errors'
 require 'json'
 require 'retriable'
-require "securerandom"
+require 'securerandom'
```

Comment:
Once we confirm that the `invocation_id` and the `idempotency-token` are same, we can remove this.